### PR TITLE
sql: clean up prettytest churn

### PR DESCRIPTION
**Pull request by kev-cao**
The golden files for the pretty tests are out of date and have not been updated in relation to their corresponding SQL files. This commit regenerates the golden files.

Epic: none

Release note: none

### CODE REVIEW

**Review by msbutler — COMMENTED** [review]
the vast majority of these changes i think live with foundations. Why hasn't anyone had to update these golden files?

File: pkg/sql/sem/tree/testdata/pretty/20.align-deindent.golden [hunk 1]
```diff
@@ -253,40 +211,53 @@ CREATE TABLE t (
...
-	a INT8, b INT8, c INT8,
-	PRIMARY KEY (a, b)
-) INTERLEAVE IN PARENT p2 (i)
-  PARTITION BY LIST (a) (PARTITION p1 VALUES IN (1), PARTITION p2 VALUES IN (2))
+79:
+-------------------------------------------------------------------------------
+CREATE TABLE t (a INT8, b INT8, c INT8, PRIMARY KEY (a, b)) PARTITION BY LIST (
+																a
+                                                            ) (
+																PARTITION p1
```
Comment (msbutler) [inline pkg/sql/sem/tree/testdata/pretty/20.align-deindent.golden:224]: this looks worse than the deleted diff. has there been a regression? I feel like foundations should confirm

File: pkg/sql/sem/tree/testdata/pretty/window.align-deindent.golden [hunk 2]
```diff
@@ -35,7 +38,7 @@ WINDOW
 			BETWEEN
 				j PRECEDING
 			AND
-				CURRENT ROW EXCLUDE NO OTHERS
```
Comment (msbutler) [inline pkg/sql/sem/tree/testdata/pretty/window.align-deindent.golden:38]: why was EXCLUDE NO OTHERS removed here?

**Comment by rafiss** [comment]
What command did you use to rewrite the files? I tried `./dev test pkg/sql/sem/tree -f=TestPrettyDataShort --test-args '-rewrite-pretty'` and there was no diff. I'm running it on commit ab2f9cfce3c87bb4f4c1382d5cc5f0905c222bf0.

**Comment by kev-cao** [comment]
@rafiss I ran `go test pkg/sql/sem/tree -rewrite-pretty` and `go test --tags=nightly pkg/sql/sem/tree -rewrite-pretty`. As you mentioned in the Slack thread, it looks like the nightly tests don't run anymore.

It seems like `go test pkg/sql/sem/tree -rewrite-pretty` runs additional tests on top of `-f=TestPrettyDataShort` because the files look different? For these prettifier golden tests, is `TestPrettyDataShort` the only one we want to test?

**Comment by msbutler** [comment]
can we close this out?

**Pull request closed without merging**